ajax/libs/ng-dropzone/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ng-dropzone",
+  "filename": "ng-dropzone.min.js",
+  "description": "AngularJS directive for __[dropzone](https://github.com/enyo/dropzone)__",
+  "homepage": "http://www.dropzonejs.com",
+  "keywords": [
+    "ng-dropzone"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/thatisuday/ng-dropzone"
+  },
+  "license": "Apache-2.0",
+  "author": "Uday Hiwarale <uhiwarale@gmail.com> (https://github.com/thatisuday)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/thatisuday/ng-dropzone",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          " *.css",
+          " *.js",
+          " *.map"
+        ]
```

Comment:
Instead of using individual file rules for each file type, you can include all files in `dist` except `.gz` files like this:

```
"files": [
  "**/*!(.gz)"
]
```